Tools-Override/tests.targets

```diff
@@ -209,6 +210,8 @@
       <TestCommandLines Include="copy /y $(TestILCFolder)\CRT\vcruntime140_app.dll %EXECUTION_DIR%\native" />
       <TestCommandLines Include="echo > %EXECUTION_DIR%\native\$(XunitTestAssembly)"/>
       <TestCommandLines Include="cd native"/>
+      <PostExecutionTestCommandLines Include="type Xunit.Console.Output.txt" />
+      <PostExecutionTestCommandLines Include="copy /y testResults.xml %EXECUTION_DIR%\" />
```

Comment:
The goal is to still move to a place were we don't need to copy and instead do all the compile and execution from the execution directory correct?